build.sbt

```diff
@@ -61,3 +61,4 @@ addSbtPlugin("com.typesafe.sbt"                          % "sbt-native-packager"
 addSbtPlugin("com.lightbend.paradox"                     % "sbt-paradox"         % "0.3.2")
 addSbtPlugin("com.geirsson"                              % "sbt-scalafmt"        % "1.4.0")
 addSbtPlugin("com.codacy"                                % "sbt-codacy-coverage" % "1.3.11")
+addSbtPlugin("io.kamon"                                  % "sbt-aspectj-runner"  % "1.0.4")
```

Comment:
we can't add this plugin to all modules as it's set to trigger automatically and adds dependencies to all modules on aspectjweaver

we need a smarter solution instead, possibly tied to the ServicePackagingPlugin